recipes/client.rb

```diff
@@ -65,20 +65,18 @@ def chef_solo_search_installed?
   rsyslog_servers += node['rsyslog']['custom_remote']
 end
 
-if rsyslog_servers.empty?
-  Chef::Application.fatal!('The rsyslog::client recipe was unable to determine the remote syslog server. Checked both the server_ip attribute and search!')
```

Comment:
We should probably turn this into an error vs. fatal, but it's a good thing to leave it present.